example/spec/parallel/pythonReferenceTests/PythonReferenceSpec.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+
+const { ActivityStep, LambdaStep } = require('@cumulus/integration-tests/sfnStep');
+const { buildAndExecuteWorkflow } = require('@cumulus/integration-tests');
+
+const { loadConfig } = require('../../helpers/testUtils');
+
+const workflowName = 'PythonReferenceWorkflow';
+
+describe('The Python Reference workflow', () => {
+  let config;
+  let lambdaStep;
+  let activityStep;
+  let workflowExecution;
+
+  const initialPayload = {
+    initialData: {
+      key1: 'value1',
+      key2: 'value2'
+    }
+  };
+
+  beforeAll(async () => {
+    config = await loadConfig();
+    process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
+
+    workflowExecution = await buildAndExecuteWorkflow(
+      config.stackName,
+      config.bucket,
+      workflowName,
+      null,
+      null,
+      initialPayload
+    );
+    lambdaStep = new LambdaStep();
+    activityStep = new ActivityStep();
+  });
+
+  it('executes successfully', () => {
+    expect(workflowExecution.status).toEqual('SUCCEEDED');
+  });
+
+  it('contains the expected output from the PythonReferenceTask', async () => {
+    const stepOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'PythonReferenceTask');
+    const expectedPayload = {
+      inputData: initialPayload.initialData,
+      configInputData: { key1: 'injectedData' },
+      newData: { newKey1: 'newData1' }
+    };
+
+    expect(stepOutput.payload).toEqual(expectedPayload);
+    expect(Object.keys(stepOutput)).toContain('meta');
```

Comment:
So these assertions just prove that the CMA is working properly and not overwriting unexpected keys?

Reply:
Yes, exactly.    We could do a more thorough evaluation of *what* is in meta/cumulus_meta, but that's not strictly constrained to this test.